Automattic/jetpack
# PHPCS: EU Cookie Widget

Fixes #21991

#### Changes proposed in this Pull Request:

PHPCS for EU Cookie Widget files.

#### Jetpack product discussion

#21991

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Linting of the affected files should produce no errors/warnings:

```sh
composer phpcs:lint projects/plugins/jetpack/modules/widgets/class-jetpack-eu-cookie-law-widget.php projects/plugins/jetpack/modules/widgets/eu-cookie-law/form.php projects/plugins/jetpack/modules/widgets/eu-cookie-law/widget.php
```

With changes pulled down, and error logs & browser console being monitored, add the Cookie & Consents widget to your site and try editing _all the things_ to make sure nothing breaks.
